src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -53,22 +53,35 @@
 
 namespace WinRT.Interop
 {
-    [ComImport]
-    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
-    [Guid("EECDBF0E-BAE9-4CB6-A68E-9598E1CB57BB")]
-    internal interface IWindowNative
+ 
+#if EMBED
+    internal 
+#else
+    public
+#endif
+    static class IWindowNativeMethods
     {
-        IntPtr WindowHandle { get; }
+        public static unsafe global::System.IntPtr get_WindowHandle(object _obj)
+        {
+            var asObjRef = global::WinRT.MarshalInspectable<object>.CreateMarshaler2(_obj, System.Guid.Parse("EECDBF0E-BAE9-4CB6-A68E-9598E1CB57BB"));

```

Comment:
we should replace this with a hardcoded static GUID like we do in other places rather than a Guid.Parse.